extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -68,7 +69,7 @@ class AmpAccordion extends AMP.BaseElement {
    */
   onHeaderClick_(event) {
     event.preventDefault();
-    const section = event.target.parentNode;
+    const section = closestByTag(event.target, 'section');
```

Comment:
This also seems unreliable - `<section>` tags can be nested freely. We may need to iterate over a list of according top-level sections (looks like they are already in `this.sections_`) and test on `section.contains(event.target)` to get the right one.